rocAL/rocAL/include/augmentations/geometry_augmentations/node_rotate.h

```diff
@@ -25,14 +25,15 @@ THE SOFTWARE.
 #include "parameter_factory.h"
 #include "parameter_vx.h"
 #include "graph.h"
+#include "rocal_api_types.h"
 
 class RotateNode : public Node
 {
 public:
     RotateNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
     RotateNode() = delete;
-    void init(float angle);
-    void init(FloatParam *angle);
+    void init(float angle, RocalResizeInterpolationType interpolation_type = ROCAL_LINEAR_INTERPOLATION);
+    void init(FloatParam* angle_param, RocalResizeInterpolationType interpolation_type = ROCAL_LINEAR_INTERPOLATION);
     unsigned int get_dst_width() { return _outputs[0]->info().max_shape()[0]; }
     unsigned int get_dst_height() { return _outputs[0]->info().max_shape()[1]; }
     vx_array get_src_width() { return _src_roi_width; }
```

Comment:
why return type is vx_array for get_src_width and unsigned int for get_dst_width?

Reply:
Because dst_width is same for all images, so it is constant
And src_width, will vary for each image hence it is an array. It is part of TOT and I think it will be used by meta nodes